release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -109,14 +117,72 @@ With granular update control, you can select a specific Operator version or defi
+Enhanced security with user-provided service accounts::
+{olmv1} prioritizes security by minimizing its permission requirements and providing greater control over access. By using xref:../extensions/ce/managing-ce.adoc#olmv1-cluster-extension-permissions_managing-ce[user-provided `ServiceAccount` objects for Operator lifecycle operations], {olmv1} access is restricted to only the necessary permissions, significantly reducing the control plane attack surface and improving overall security. In this way, {olmv1} adopts a least-privilege model to minimize the impact of a compromise.
+
+[NOTE]
+====
+The documentation for {olmv1} exists as a stand-alone guide called xref:../extensions/index.adoc#olmv1-about[Extensions]. Previously, {olmv1} documentation was a subsection of the xref:../operators/index.adoc#operators-overview[Operators] guide, which otherwise documents the {olmv0} feature set.
+
+The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
+====
 
 [id="ocp-4-18-extensions-supported-extensions_{context}"]
-==== {olmv1} supported extensions and known issue
+==== {olmv1} supported extensions
 
 include::snippets/olmv1-tp-extension-support.adoc[]
 
+[id="ocp-4-18-extensions-disconnected_{context}"]
+==== Disconnected environment support in {olmv1}
+
+To support cluster administrators that prioritize high security by running their clusters in internet-disconnected environments, especially for mission-critical production workloads, {olmv1} includes cluster extension lifecycle management functionality that works within these disconnected environments, starting in {product-title} {product-version}.
+
+After using the oc-mirror plugin for the {oc-first} to mirror the images required for your cluster to a mirror registry in your fully or partially disconnected environments, {olmv1} can function properly in these environments by utilizing the sets of resources generated by either oc-mirror plugin v1 or v2.
+
+For more information, see xref:../extensions/catalogs/disconnected-catalogs.adoc#disconnected-catalogs[Disconnected environment support in {olmv1}].
+
+[id="ocp-4-18-extensions-catalog-selection_{context}"]
+==== Improved catalog selection in {olmv1}
+
+With this release, you can perform the following actions to control the selection of catalog content when you install or update a cluster extension:
+
+* Specify labels to select the catalog
+* Use match expressions to filter across catalogs
+* Set catalog priority
+
+For more information, see xref:../extensions/catalogs/catalog-content-resolution.adoc#catalog-content-resolution[Catalog content resolution].
+
+[id="ocp-4-18-extensions-proxy-and-trustedCA_{context}"]
+==== Basic support for proxied environments and trusted CA certificates
+
+With this release, Operator Controller and catalogd can now run in proxied environments and include basic support for trusted CA certificates.
```

Comment:
cc @tmshort . I think we do not have support for trusted CA in 4.18. But Todd can confirm.

Reply:
The projected volume for the trusted CA is there. The only limitation is that if any of the proxy environment variables change, we won't see it, cluster-olm-operator would have to be restarted, in order to re-do the deployment

Reply:
Cancel that... the proxy informer is there